httpmw/log.go

```diff
@@ -48,7 +49,10 @@ func (mid *logHandler) ServeErrHTTP(w http.ResponseWriter, r *http.Request) erro
 		slog.Any("network", ddlog.NewNetworkMap(r)))
 	if err != nil {
 		var errLeveler slog.Leveler
-		if errors.As(err, &errLeveler) {
+		if errors.Is(err, context.Canceled) {
+			log = log.With(slog.Bool("canceled", true))
+			// and keep the info level regardless of severity
```

Comment:
`DeadlineExceeded` probably makes sense to keep logged as error, since, compared to `Canceled`, it was not canceled intentionally. I'd make it a bit more obvious that `Canceled` doesn't need to be logged as error because it (likely?) was intentional.